### ISSUE TITLE
Song Download failed

### ISSUE THREAD

**Issue by SnesLover22** [111116/webosu#94]
So I was able to play today, and like 3-4 hours later I got a message that it failed to download the songs. Is it because the server is down (I made a fork of this repo and it displays that message, and the other one, https://osu.ukenn.top/ doesn't have that message)? Is it on my end?

 - OS: [Chromeos]
 - Browser [Chrome]
 - Version [?]

**Comment by JMcrafter26** [comment]
> So I was able to play today, and like 3-4 hours later I got a message that it failed to download the songs. Is it because the server is down (I made a fork of this repo and it displays that message, and the other one, https://osu.ukenn.top/ doesn't have that message)? Is it on my end?
> 
> * OS: [Chromeos]
> * Browser [Chrome]
> * Version [?]

Hi there, 
I have the exact same issues.
My conclusion is, that the server providing the Beat map information are currently unviable.

In my opinion You cannot do anything of this, you need to wait… 🙄

If you could solve this issue, please let me know

**Comment by SnesLover22** [comment]
aw man! this is dissapointing :( i did find another site: https://osu.michael.kim/ and it has an option to add beatmaps locally and also from servers, including Sayobot. The other one is ripple. I suggest to the creator he add Ripple as one of the servers. Seems more reliable to have 2

**Comment by JMcrafter26** [comment]
> aw man! this is dissapointing :( i did find another site: https://osu.michael.kim/ and it has an option to add beatmaps locally and also from servers, including Sayobot. The other one is ripple. I suggest to the creator he add Ripple as one of the servers. Seems more reliable to have 2

Yes, it would be great to have one backup server. But I don't think the creator would update it, because it was last updated a year ago 😕

**Comment by SnesLover22** [comment]
dang. maybe someone else can take over. I did make a fork but idk much about how they did it/ using JS.

**Comment by JMcrafter26** [comment]
Hi there, the Problem See s to be gone. 
Maybe this was a temporary issue.
You can closer this issue now.

Happy Coding 🎈

**Comment by SnesLover22** [comment]
yep I saw it was down :(

**Comment by SnesLover22** [comment]
noice!

**Comment by JMcrafter26** [comment]
> yep I saw it was down :(

Seems to be up again

**Comment by SnesLover22** [comment]
Yeah i know! yay so happy! maybe if the mirror site is on github you can make a fork of that so it never goes down again? idk

**Comment by safewallaby** [comment]
> aw man! this is dissapointing :( i did find another site: https://osu.michael.kim/ and it has an option to add beatmaps locally and also from servers, including Sayobot. The other one is ripple. I suggest to the creator he add Ripple as one of the servers. Seems more reliable to have 2

Do you know any other osu websites?

**Comment by JMcrafter26** [comment]
> > aw man! this is dissapointing :( i did find another site: https://osu.michael.kim/ and it has an option to add beatmaps locally and also from servers, including Sayobot. The other one is ripple. I suggest to the creator he add Ripple as one of the servers. Seems more reliable to have 2
> 
> Do you know any other osu websites?

There are some websites out there, just Google for "osu! Beatmap download". 

The website this repo is using is called [https://osu.sayobot.cn/](https://osu.sayobot.cn/)

**Comment by SnesLover22** [comment]
No. I do know another one but its trash. https://osu-web.com/ . also the site (osu.ukenn.top) is down again

**Comment by JMcrafter26** [comment]
> No. I do know another one but its trash. https://osu-web.com/ . also the site (osu.ukenn.top) is down again

[https://www.webosu.online/](https://www.webosu.online/) seems to be working

**Comment by SnesLover22** [comment]
well that one is blocked for me

**Comment by safewallaby** [comment]
@SnesLover22 i found the best one so far https://joshua-usi.github.io/osw/

**Comment by cheemzburgs** [comment]
> @SnesLover22 i found the best one so far https://joshua-usi.github.io/osw/

levels will not load when opened
